azurerm/resource_arm_virtual_machine.go

```diff
@@ -647,6 +675,10 @@ func resourceArmVirtualMachineRead(d *schema.ResourceData, meta interface{}) err
 		}
 	}
 
+	if resp.Identity != nil {
+		d.Set("identity", flattenAzureRmVirtualMachineIdentity(resp.Identity))
+	}
```

Comment:
apologies, I've just noticed I didn't post the output of the acceptance tests in here. After running the VM acceptance tests - the acceptance tests without this field specified currently fail:

```
------- Stdout: -------
=== RUN   TestAccAzureRMVirtualMachine_tags
--- FAIL: TestAccAzureRMVirtualMachine_tags (416.59s)
    testing.go:459: Step 0 error: After applying this step, the plan was not empty:
        
        DIFF:
        
        UPDATE: azurerm_virtual_machine.test
          identity.#: "" => "<computed>"
        
        STATE:
...
```

In order to work around this - the field needs to be set to an empty object when there's no Identity returned - as such I've pushed a commit to fix this

Reply:
I've re-run all the VM tests and this now looks good - apologies for the delayed re-review here!